haystack/pipeline.py

```diff
@@ -593,6 +596,39 @@ def run(self, **kwargs):
         return kwargs, "output_1"
 
 
+class QueryClassifier(BaseComponent):
+    """
+    A node to choose between two nodes based on query classification result.
+    """
+
+    outgoing_edges = 2
+
+    def __init__(
+        self, query_classifier: GradientBoostingClassifier, query_vectorizer: TfidfVectorizer
```

Comment:
How about making query_classifier and query_vectorizer optional. And initializing by default (if user doen't pass these params) using respective s3 hosted pickle files.

Reply:
@lalitpagaria sound, good. Let me do that.

Reply:
@lalitpagaria does it look better now?